packages/shared/src/components/feeds/FeedSettings/FeedSettingsEdit.tsx

```diff
@@ -65,6 +68,16 @@ export const FeedSettingsEdit = ({
     },
   ];
 
+  useEffect(() => {
+    if (!isPlus) {
+      router.replace(webappUrl);
```

Comment:
This is existing feed, but not plus member/enrolled?

Reply:
basically if you are not plus you can't edit